### PR TITLE
React Type Declarations

### PR DESCRIPTION
**Pull request by Conner1115**
React type declarations work now!!!!!!!!!!!!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
     "./react": {
-      "types": "./dist/react/index.d.ts",
-      "import": {
-        "types": "./dist/react/index.d.ts",
-        "default": "./dist/react/index.js"
-      },
-      "default": "./dist/react/index.cjs"
+      "import": "./dist/react/index.js",
+      "require": "./dist/react/index.cjs",
+      "types": "./dist/react"
```
Comment (lunaroyster) [inline package.json:15]: have we made sure both `require` and es6 `import` works?

Comment (lunaroyster) [reply package.json:15]: I remember arriving at that structure after a lot of iteration so that every type of import works

Comment (Conner1115) [reply package.json:15]: I did a lot of research, and that is how you do it.  I haven't tested it out, but theoretically it should work.

If we need to test it out, how would we do that?

Comment (Conner1115) [reply package.json]: Added a `default` export to the list, nothing should be missed now

**Review by lunaroyster — APPROVED** [review]
thank you so much!

**Pull request merged**